libs/template/config.go

```diff
@@ -99,9 +99,38 @@ func (c *config) assignDefaultValues(r *renderer) error {
 	return nil
 }
 
+func (c *config) isSkipped(p jsonschema.Property) bool {
+	if p.Schema.SkipPromptIf == nil {
+		return false
+	}
+
+	// Check if conditions for skipping are satisfied
+	skip := true
+	for name, property := range p.Schema.SkipPromptIf.Properties {
+		if v, ok := c.values[name]; ok && v == property.Const {
+			continue
+		}
```

Comment:
What about skips on patterns? Are constants enough for the planned use cases?

Reply:
To start with, yes. The only planned change I am aware of is gating `input_schema_name` behind `input_include_models_in_unity_catalog` and maybe some general gating of UC specific questions behind a broader is_uc prompt. 

Support for patterns can be added once there's a requirement for it from the mlops-stacks folks.

Reply:
schema: https://github.com/databricks/mlops-stacks/blob/main/databricks_template_schema.json